hooks/formatter

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-for file in $(find $(git rev-parse --show-toplevel) | egrep "\.(c|cc|cpp|h|hh|hpp|m|mm)\$" | egrep -v "third_party|output_base");
+for file in $(find $(git rev-parse --show-toplevel) | egrep "\.(c|cc|cpp|h|hh|hpp|m|mm)\$" | egrep -v "third_party|output_base|legato");
```

Comment:
Create a new file named `format-exclude-list`, which contains the directories/files you tend to exclude. Always make scripts easier to maintain.